DeviceClassifier/README.md

```diff
@@ -41,10 +41,10 @@ export PCAP=[path/to/file.pcap]
 make run
 ```
 
-And an example of explicitly calling `eval_onelayer`:
+And an example of explicitly calling `test_onelayer`:
 ```
-export PCAP=[path/to/file.pcap]
-make eval_onelayer
+export PCAP=[path/to/pcapdir]
```

Comment:
shouldn't this still be just a file not a dir?

Reply:
Not for test, tho, right?

Reply:
ah never mind, lemme change that.